spec/toy_robot_spec.rb

```diff
@@ -12,24 +12,32 @@
       expect(subject.y).to eq(0)
     end
 
-    specify 'that ToyRobot f is set to EAST' do
-      expect(subject.f).to eq(:east)
+    specify 'that ToyRobot facing is set to NORTH' do
+      expect(subject.facing).to eq(:north)
     end
   end
 
   describe 'place' do
     subject{ ToyRobot.new }
-    specify 'When x is less than zero it should raise an exception' do
-      expect { subject.place(-1,0) }.to raise_error(InvalidPositionError, "Invalid Position, -1, 0")
+    specify 'When x is less than zero, it expect to raise an exception' do
+      expect { subject.place(-1,0,:north) }.to raise_error(InvalidPositionError, "Invalid Position, -1, 0, north")
```

Comment:
Should I set the default to :east and test it with valid value (3, 4, :west) back again?